docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx

```diff
@@ -69,16 +76,20 @@ Else, you can skip this step:
 
 ```code
 $ sudo touch /etc/teleport-upgrade.d/schedule
-$ sudo chown <your-teleport-user> /etc/teleport-upgrade.d/schedule
+$ sudo chown <Var name="your-teleport-user" /> /etc/teleport-upgrade.d/schedule
```

Comment:
I thought you couldn't use vars in code blocks?

Reply:
You can.  You can't in other types like `yaml`. 

![image](https://github.com/gravitational/teleport/assets/60704961/fc811f5e-52d3-493f-8670-55f062549161)